assets/scss/_layout.scss

```diff
@@ -80,3 +80,7 @@
   display: flex;
   flex-direction: column;
 }
+
+.w-full {
```

Comment:
Please don't use short names, like the rest of the CSS. This could be `width-full` or `full-width`